cell_computing_service.py

```diff
@@ -32,17 +36,16 @@ def ComputeCellInteractions(self, incoming_batch, context):
         for c in incoming_batch.cells_to_compute:
             cis_env.move_cell_and_connected_cells(
                 c, id_to_cell, id_to_cell_moved)
+
         # Interaction
 
-        # Energy
+        # Get Energy
         for c in incoming_batch.cells_to_compute:
-            cis_env.feed_cell(
-                c,
-                id_to_cell,
-                id_to_cell_energy_averaged,
-                food_factor=conf.WANTED_CELL_AMOUNT_PER_BUCKET /
-                len(
-                    incoming_batch.cells_to_compute))
+            c = cis_env.feed(c, incoming_batch.time_step)
+
```

Comment:
FYI: After feeding all the cells, it is probably a good idea to still average the energy level inside a connected cell group with `average_out_energy_in_connected_cells` (you removed that call in the merge)